app/build.gradle

```diff
@@ -98,8 +103,9 @@ dependencies {
     implementation "com.google.dagger:dagger-android-support:2.25.2"
     kapt "com.google.dagger:dagger-compiler:2.25.2"
     kapt "com.google.dagger:dagger-android-processor:2.25.2"
+    implementation 'com.google.android.libraries.places:places:2.3.0'
```

Comment:
You still have the `:places` dependency duplicated. Also they're 2 different versions `2.3.0` and `2.2.0`. Pick one of them and remove other